async_simple/coro/test/LazyTest.cpp

```diff
@@ -420,7 +420,7 @@ TEST_F(LazyTest, testCollectAll) {
 TEST_F(LazyTest, testCollectAllBatched) {
     size_t task_num = 5000;
     long total = 0;
```

Comment:
```long total``` --> ```size_t total``` may be better.
Maybe unified all int to size_t can get ride of static_cast.

Reply:
make task_num from `size_t` to `int`.